conda.recipe/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: datashader
+  version: 0.1.0
+
+extra:
+  channels:
+    - blaze
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - dask >=0.7.6
+    - datashape >=0.5.1
+    - numba >=0.23.1
+    - odo >=0.4.2
+    - numpy >=1.7
+    - pandas >=0.15.0
+    - pillow >=3.1.1
+    - pytest >=2.8.5
```

Comment:
Sorry, another comment :/. Pytest isn't a hard requirement for running, but should be a build requirement (since tests are run on build). Is this correct?